Makefile

```diff
@@ -38,7 +38,7 @@ endif
 .PHONY: py-test
 py-test:
 	@echo "Running python tests"
-	pytest --reuse-db --cov --cov-report term:skip-covered
+	pytest --ds=hypha.settings.test --reuse-db --cov --cov-report term:skip-covered
```

Comment:
The `--ds=...` is picked from the `setup.cfg` https://github.com/HyphaApp/hypha/blob/main/setup.cfg#L23 by pytest; this need not be duplicated here. Can you double-check it?

Reply:
Will remove it.